releng/raw-binaries/Dockerfile

```diff
@@ -43,11 +49,19 @@ RUN mkdir -p /opt/osxcross && \
 
 ENV PATH="/opt/osxcross/target/bin:${PATH}"
 
-# Wrapper to pull a version of pkg-config at runtime
-ENV PKG_CONFIG=/usr/local/bin/pkg-config.sh
+RUN cd /tmp && \
+    git clone https://github.com/richfelker/musl-cross-make.git && \
+    cd musl-cross-make && \
+    git checkout v0.9.9 && \
+    make MUSL_VER=1.1.24 TARGET=aarch64-unknown-linux-musl DL_CMD="curl -C - -L -o" install && \
```

Comment:
Do we know what we are curling with this command? Is there any way we can place constraints on it?

Reply:
I copied this from the 2.x build - more safety would be awesome.